website/static/js/pages/project-settings-page.js

```diff
@@ -245,6 +245,8 @@ $(document).ready(function() {
                     callback: function(result) {
                         if (!result) {
                             $(that).attr('checked', false);
+                        } else {
+                            $('#selectAddonsForm').submit();
```

Comment:
To me, calling the selector seems unnecessary.  I might change the jQuery submit listener into a function.  

something like   
```
function submitBucket() {
```

then call submitBucket()